test_doubly_linkedlist.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+
+from doubly_linkedlist import Dll
+
+
+def test_dll():
+    dll = Dll([4, 6, 8, 2])
+
+    assert dll.head.value == 2
+    assert dll.head.next.value == 8
```

Comment:
In this entire file, if ti is worth checking what `.next.value` is, it is also worth checking what `.previous.value` is.  Obviously in some cases previous is None rather than having a .value.  But that should only be the head.